fact_extractor/plugins/unpacking/sevenz/install.sh

```diff
@@ -1,22 +1,29 @@
 #!/usr/bin/env bash
+set -e
 
 cd "$( dirname "${BASH_SOURCE[0]}" )"
 
 echo "------------------------------------"
 echo "      install p7z from source       "
 echo "------------------------------------"
 
-sudo -EH pip3 install --upgrade git+https://github.com/fkie-cad/common_helper_passwords.git
 
 # install newest version of p7zip
 sudo apt-get remove -y p7zip-full
 
-mkdir /tmp/fact_build
+mkdir /tmp/fact_build || :
 cd /tmp/fact_build
 
 wget -O 7zip.tar.bz2 https://sourceforge.net/projects/p7zip/files/latest/download
 tar xvjf 7zip.tar.bz2
-(cd p7zip* && cp makefile.linux_amd64_asm makefile.linux && make -j$(nproc) all3 && sudo ./install.sh)
+cd p7zip*
+# gcc >= 11 has -Wnarrowing as default flag which leads to an error during compilation
+# g++ will try to use standard C++17 but the code is not compatible -> use C++14
+sed -i 's/CXXFLAGS=-c -I. \\/CXXFLAGS=-c -I. -Wno-narrowing -std=c++14 \\/g' makefile.glb  || echo "Warning: Could not apply makefile patch"
```

Comment:
Good thing we have tests...

Reply:
It was really fun debugging this :facepalm:

Reply:
The question is: should we pin the 7z version so that this will not cause a problem with a new version?